interop/async/main.go

```diff
@@ -0,0 +1,54 @@
+
+package main
+
+/*
+	#cgo LDFLAGS: -L./ -lasynclib
+
+	#include <stdio.h>
```

Comment:
```suggestion
	#include <stddef.h>
```

Reply:
Thanks for the comment @arnetheduck ! However, if I apply this change, I get the next error:
> ./main.go:46:5: could not determine kind of name for C.getchar